Tests/Services/NfieldSurveyInviteRespondentsServiceTests.cs

```diff
@@ -120,6 +120,60 @@ private static bool FilterEquals(SampleFilter filter, string name, string op, st
 
         #endregion
 
+        [Fact]
+        public void TestGetStatusAsync_SurveyIdIsNull_Throws()
+        {
+            var target = new NfieldSurveyInviteRespondentsService();
+
+            Assert.Throws<ArgumentNullException>(() =>
+                UnwrapAggregateException(target.GetInvitationStatusAsync(null, null)));
+        }
+
+        [Fact]
+        public void TestGetStatusAsync_SurveyIdIsEmpty_Throws()
+        {
+            var target = new NfieldSurveyInviteRespondentsService();
+
+            Assert.Throws<ArgumentException>(() =>
+                UnwrapAggregateException(target.GetInvitationStatusAsync("  ", null)));
```

Comment:
Actually, this is not testing if the survey id is empty, but if it is a white space. Replace the GetInvitationStatusAsync("  ", null) by GetInvitationStatusAsync(string.Empty, null) or rename the test